hail/python/hail/ggplot/ggplot.py

```diff
@@ -206,6 +206,14 @@ def increment_and_return_old():
         if self.coord_cartesian is not None:
             self.coord_cartesian.apply_to_fig(fig)
 
+        fig = fig.update_xaxes(title_font_size=18)
+        fig = fig.update_yaxes(title_font_size=18)
+        fig = fig.update_layout(
+            template="simple_white",
+            font_family="Ariel",
```

Comment:
what's the exact syntax here? If a system doesn't have Ariel, what will happen?

Reply:
Okay, just looked it up, the font stack used by default is: `'"Open Sans", verdana, arial, sans-serif'`, this should be good enough for all cases, if you'd like, youcan reorder it to prefer Arial, but we definitely should not be deleting sans-serif as a fallback to the system sans-serif in case only basic system fonts are installed.

Reply:
Really good catch, will fix.